### ISSUE TITLE
StackOverflowException in ArrayUtils for custom types

### ISSUE THREAD

**Issue by kdebski85** [killme2008/aviatorscript#484]
ArrayUtils class throws StackOverflowException when used with an array of any custom type, for example:`ArrayUtils.getLength(new Foo[]{})`.

The issue is that `getLength` method calls itself as a fallback:

```java
  public static int getLength(final Object a) {
   (...)

    return ArrayUtils.getLength(a);
  }
```

The line `return ArrayUtils.getLength(a);` in this method should be replaced with `return Array.getLength(a);`

The same issue is for `get` and `set` methods in this class.

**Comment by killme2008** [comment]
感谢，这是一个 bug，会尽快修复

**Comment by kdebski85** [comment]
Thank you for the fix. I hope 5.3.2 will be released soon. I am closing this issue.

**Comment by killme2008** [comment]
Fixed in 5.3.2 https://github.com/killme2008/aviatorscript/releases/tag/aviator-5.3.2
